### PR TITLE
Merge `8.2.x` into `develop`

### PR DESCRIPTION
**Pull request by boherm**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Merge `8.2.x` into `develop`
| Type?             | improvement
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI + Tests UI
| UI Tests          | https://github.com/boherm/ga.tests.ui.pr/actions/runs/11103487385
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Comment by ps-jarvis** [comment]
This pull request seems to contain new translation strings. I have summarized them below to ease up review:

- `Admin.Modules.Notification`
    - [ ] `The override file %1$s conflicts with an existing override in %2$s.`

(Note: this is an automated message, but answering it will reach a real human)

**Review by nicosomb — APPROVED** [review]

**Review by nicosomb — DISMISSED** [review]
CI is 🔴

File: controllers/admin/AdminCartsController.php
Comment (jolelievre) [inline controllers/admin/AdminCartsController.php:1]: This controller has been migrated so this file should not be integrated back in the develop branch and must be deleted

File: src/PrestaShopBundle/Resources/config/services/core/module.yml
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/config/services/core/module.yml]: We should keep the 8.1 indication to know when this was deprecated, so we can remove it in the next version
Actually, this should be removed in the v9.0 but in a dedicated PR probably

File: translations/default/ModulesBlockreassuranceAdmin.xlf
Comment (jolelievre) [inline translations/default/ModulesBlockreassuranceAdmin.xlf]: You should modify absolutely NO translation files, you can revert the changes on all those files

**Review by jolelievre — CHANGES_REQUESTED** [review]

File: classes/Customer.php
Comment (Hlavtox) [inline classes/Customer.php]: Please don't merge anything in this method, it's already fixed on develop branch.

**Review by Hlavtox — CHANGES_REQUESTED** [review]
See comment, please do not change Customer.php.

**Review by Hlavtox — DISMISSED** [review]
All good now. :-) thanks @boherm

**Review by matthieu-rolland — DISMISSED** [review]
thanks @boherm 👍

File: src/PrestaShopBundle/Controller/Admin/Improve/Design/CmsPageController.php
Comment (jolelievre) [inline src/PrestaShopBundle/Controller/Admin/Improve/Design/CmsPageController.php:95]: Lucky for you it was not refactored yet ^^

File: tests/Integration/Core/Addon/Module/ModuleManagerBuilderTest.php
Comment (jolelievre) [inline tests/Integration/Core/Addon/Module/ModuleManagerBuilderTest.php]:
```suggestion
            'DummyAdminController.php file different'
```

**Review by jolelievre — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
